Dispatchfile

```diff
@@ -111,6 +111,7 @@ task("update-soak", inputs=[git_resource_konvoy_soak, git_resource_kubeaddons_ko
 # Dispatch Actions
 
 action(tasks=["update-soak"], on=p.Condition(release=p.ReleaseCondition(tags=["v*"])))
+action(tasks=[update_konvoy_task], on=p.Condition(release=p.ReleaseCondition(tags=["v*"])))
```

Comment:
We might want to consider doing the regex reading for `testing|stable` here as opposed to the python script.

Reply:
`testing.*` is not official and should not be used.

Reply:
So we don't want to create a pull request for it against konvoy? Only for `stable.*`?

Reply:
I think that may be true. @dkoshkin ?

Reply:
> should do the regex reading for testing|stable here as opposed to the python script.

But the tag in which it is published is different than `testing|stable` branch.   In Python script, 

1) `testing|stable` is determined from the _previous_ testing|stable already present in the Makefile
2) We pick the latest testing/stable and update the Makefile.

This event might not be available in `tags' because tags look like `v123` (and branch corresponds to stable|testing dependency on the target branch of konvoy). We rely on the Makefile in the target branch to give us the previous version we want, and bump to higher version.

(_hope what I explained is understable_) - I am open to changing approaches - whichever is more accurate.

Reply:
@joejulian - This will create PR against when both testing or stable is updated / push. Merging the PR will be our choice at the konvoy end. The integrating via PR, and testing it, I believe, is still useful.